Src/AutoFixture/PostprocessComposerExtensions.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Linq.Expressions;
+using AutoFixture.Dsl;
+
+namespace AutoFixture
```

Comment:
Can we move it under the `Dsl` namespace? I believe that nowadays IDEs perform discovery well even for non-imported namespaces, so it should not be an issue.

Reply:
Speaking about Visual Studio, whilst it's able to find an extension method if you use the correct types, I don't think that would be the case here because we use open types.

My fear is that these extension methods are difficult to be discovered for their "shape", from the user if not from the IDE. Having them readily available in the IntelliSense menu would mitigate the issue a bit.